lib/install.ps1

```diff
@@ -34,9 +34,14 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
-    if ((get_config 'manifest-review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
-        Write-Output "Manifest: $app.json"
-        Write-Output $manifest | ConvertToPrettyJson
+    if ((get_config 'manifest_review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
```

Comment:
```suggestion
    if ((get_config 'manifest-review' $false) -and ($MyInvocation.ScriptName -notlike '*scoop-update*')) {
```

Reply:
This is a bug. The value should be `manifest_review`. See https://github.com/ScoopInstaller/Scoop/blob/d1f828c94275641b86838e90a041af3315fb7d0f/libexec/scoop-config.ps1#L59

Reply:
Aha, yes